lib/OpenQA/WebAPI/Controller/API/V1/Job.pm

```diff
@@ -253,6 +253,52 @@ sub update_status {
     $self->render(json => $ret);
 }
 
+sub update {
+    my ($self) = @_;
+
+    my $resultset = $self->app->schema->resultset('Jobs');
+    my $job       = $resultset->find(int($self->stash('jobid')))
+      or return $self->render(json => {error => 'Job does not exist'}, status => 404);
```

Comment:
OK, I don't mind if you go with this code pattern - if you change all other routes at least in this file (in an extra commit to stay in peace with @okurz :). But this is about the 5th introduction into an action - only in this Controller.